crates/storage/provider/src/traits/block.rs

```diff
@@ -34,15 +33,17 @@ impl StorageLocation {
 #[auto_impl::auto_impl(&, Arc, Box)]
 pub trait BlockExecutionWriter: BlockWriter + Send + Sync {
     /// Take range of blocks and its execution result
-    fn take_block_and_execution_range(
+    fn take_block_and_execution_range_above(
```

Comment:
range_above doesn't make much sense to me

Reply:
updated